packages/gatsby-theme-aio/src/components/GetCredential/Return/ReturnImsOrgID.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import ShowCard from '../Card/ShowCard';
+
+const ReturnImsOrgID = ({ returnImsOrgID , imsOrgId , value}) => {
+  return <ShowCard heading={returnImsOrgID?.heading} value={value} />;
+};
+
+export { ReturnImsOrgID };

```

Comment:
Duplicate of `packages/gatsby-theme-aio/src/components/GetCredential/Card/CardImsOrgID.js`. Can we merge them into one?

Reply:
Okay